services/app-web/src/pages/SubmissionSideNav/SubmissionSideNav.tsx

```diff
@@ -103,15 +103,19 @@ export const SubmissionSideNav = () => {
 
     const isCMSUser = loggedInUser?.role === 'CMS_USER'
     const isAdminUser = loggedInUser?.role === 'ADMIN_USER'
+    const isHelpdeskUser = loggedInUser?.role === 'HELPDESK_USER'
 
     // CMS Users can't see DRAFT, it's an error
-    if (submissionStatus === 'DRAFT' && (isCMSUser || isAdminUser)) {
+    if (
+        submissionStatus === 'DRAFT' &&
+        (isCMSUser || isAdminUser || isHelpdeskUser)
```

Comment:
Curious if a help desk user would ever be asked for support on a draft submission.

Reply:
Yes, they would, I'm going to file a ticket to track that. We should also make sure they can't pose questions etc. The goal with this PR is to get some access to our HD users quickly.